source/about/mobile-app-changelog.md

```diff
@@ -5,6 +5,31 @@
 
 This changelog summarizes updates to Mattermost mobile apps releases for [Mattermost](https://mattermost.com).
 
+(release-v2-25-0)=
+## 2.25.0 Release
+ - Release Date: February 14, 2025
+ - Server Versions Supported: Server v9.11.0+ is required. Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device.
+
+### Compatibility
+ - **Upgrade to server version v9.11.0 or later is required.** Support for server [Extended Support Release](https://docs.mattermost.com/about/release-policy.html#extended-support-releases) (ESR) v9.5.0 has ended and upgrading to server ESR v9.11.0 or later is required. As we innovate and offer newer versions of our mobile apps, we maintain backwards compatibility only with supported server versions. Users who upgrade to the newest mobile apps while being connected to an unsupported server version can be exposed to compatibility issues, which can cause crashes or severe bugs that break core functionality of the app.
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).	
+ - iPhone 6s devices and later with iOS 13.4+ are required.
+
+### Breaking Changes
+ - In the Mattermost Mobile App v2.25 release, Mattermost has stopped supporting iOS versions 13 and 14. Users should update their iOS version to v15.1 or newer before February 14, 2025. See more details in [this forum post](https://forum.mattermost.com/t/deprecation-notice-ios-13-and-14-versions/21845).
+
+### Improvements
+ - Added a feature for having a separate tab for **Drafts** on mobile.
+ - Added visualization of custom profile attributes in the bottom sheet.
```

Comment:
```suggestion
 - Added visualization of custom profile attributes. Requires server v.10.5 and feature flag enabled.
```
Not sure if we have referenced FF in the past, but to clarify this won't work just by updating the app. @amyblais feel free to adapt to the usual phrasing